src/main/resources/com/microsoft/jenkins/containeragents/aci/AciContainerTemplate/config.jelly

```diff
@@ -49,6 +49,13 @@
                                     deleteCaption="${%Delete_Volume}"/>
     </f:entry>
 
+
+    <f:dropdownList selectedName="ipType" title="${%IP_Type}" name="ipType">
```

Comment:
screenshot? probably simpler to do this with a checkbox?

`Use private instance` or something like that?

Reply:
I replaced it by a checkbox "Use private IP Address"

![grafik](https://user-images.githubusercontent.com/976007/146353301-da2c9902-43cf-4b16-8381-f483bababf13.png)